sycl/test/basic_tests/vectors/swizzle_aliases.cpp

```diff
@@ -0,0 +1,17 @@
+// RUN: %clangxx -fsycl -fsyntax-only %s
+
+#include <sycl/sycl.hpp>
+
+int main() {
+  sycl::queue Q;
+  Q.single_task([]() {
+    sycl::vec<int, 4> X{1};
+    static_assert(std::is_same_v<decltype(X.swizzle<0>())::element_type, int>);
+    static_assert(std::is_same_v<decltype(X.swizzle<0>())::value_type, int>);
+#ifdef __SYCL_DEVICE_ONLY__
+    static_assert(std::is_same_v<decltype(X.swizzle<0>())::vector_t,
+                                 decltype(X)::vector_t>);
```

Comment:
Is this right?  `X.swizzle<0>()` is a 1-element vector while `X` is a 4-element vector.  Are the `vector_t` aliases the same regardless of the vector width?

Reply:
Right you are! I was a little too quick with that one. It has been fixed.